arch/arm64/boot/dts/qcom/msm8953-xiaomi-vince.dts

```diff
@@ -301,8 +301,9 @@
 &modem {
 	status = "okay";
 
+	firmware-name = "qcom/sdm625/mba.mbn", "qcom/sdm625/modem.mdt";
```

Comment:
This is an unrelated change, so at least make a separate commit from this.

Reply:
Maybe just remove this line? I think if it required, it should be present in all the dts, but it is just firmware path

Reply:
Yeah a mistake. cherry-picking the commit from another branch of mine